src/components/tx/SignOrExecuteForm/UnknownContractError.tsx

```diff
@@ -0,0 +1,31 @@
+import { type ReactElement } from 'react'
+import { ImplementationVersionState } from '@safe-global/safe-gateway-typescript-sdk'
+import ExternalLink from '@/components/common/ExternalLink'
+import { useCurrentChain } from '@/hooks/useChains'
+import useSafeInfo from '@/hooks/useSafeInfo'
+import { getExplorerLink } from '@/utils/gateway'
+import ErrorMessage from '../ErrorMessage'
+
+const UnknownContractError = (): ReactElement | null => {
+  const { safe, safeAddress } = useSafeInfo()
+  const currentChain = useCurrentChain()
+
+  // Unsupported base contract
+  const isUnknown = safe.implementationVersionState === ImplementationVersionState.UNKNOWN
```

Comment:
Whilst working on a separate refactor, I found a function we can reuse here: `isValidMasterCopy` in `@/services/contracts/safeContracts.ts`